### DISCUSSION TITLE
Any plans to make Docker Images publicly available?

### DISCUSSION THREAD

**Discussion by dkindlund** [FlowiseAI/Flowise#3201] [Ideas]
Rather than having to build your own Docker image every time there is a new version, @HenryHengZJ do you have any plans on creating Docker images for the community to use?  I'm thinking about something like this: https://github.com/n8n-io/n8n/pkgs/container/n8n -- for example.  Thanks!

**Comment by HenryHengZJ** [comment]
We have dockerhub image here https://hub.docker.com/r/flowiseai/flowise/tags

**Reply by dkindlund** [reply]
Wonderful!  Thanks for the heads up.
